proxy/src/test/resources/com/wavefront/agent/preprocessor/preprocessor_rules.yaml

```diff
@@ -395,3 +395,15 @@
     action        : spanAddTag
     key           : multiPortSpanTagKey
     value         : multiSpanTagVal
+
+  ## obfuscate Tests
+  - rule    : example-obfuscation
+    action  : obfuscate
+    key     : 1234567890123456
+    scope   : metricName
```

Comment:
It seems like metricName is a key word scope; so you would need matching parameters with regex pattern like
- rule    : example-force-lowercase
    action  : forceLowercase
    scope   : metricName
    match   : "^UPPERCASE.*$"

https://docs.wavefront.com/proxies_preprocessor_rules.html#forcelowercase